api/envoy/config/core/v3/base.proto

```diff
@@ -346,9 +346,10 @@ message HeaderValueOption {
 
   // Describes the supported actions types for header append action.
   enum HeaderAppendAction {
-    // This action will append the specified value to the existing values if the header
-    // already exists. If the header doesn't exist then this will add the header with
-    // specified key and value.
+    // If the header already exists, this action will result in:
+    // - Comma concatenated for predefined inline headers.
+    // - Duplicate header added in the HeaderMap for other headers.
+    // If the header doesn't exist then this will add new header with specified key and value.
```

Comment:
```suggestion
    // If the header already exists, this action will result in:
    //
    // - Comma concatenated for predefined inline headers.
    // - Duplicate header added in the HeaderMap for other headers.
    //
    // If the header doesn't exist then this will add new header with specified key and value.
```

Reply:
done, thanks.